hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -1,8 +1,7 @@
 // Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
-
-
+//
```

Comment:
There seems to be a lot of random noise in this PR that seems unrelated to the change?

Reply:
It is related. Flash ctrl stuff is autogenerated with another script which has not been updated.  This should get cleaned up in subsequent PRs.